src/components/track-tile/TrackTileSkeleton.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import { ImageStyle, StyleSheet, View } from 'react-native'
+
+import DynamicImage from 'app/components/dynamic-image'
+import Skeleton from 'app/components/skeleton'
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+
+import { TrackTileBottomButtons } from './TrackTileBottomButtons'
+import { TrackTileContainer } from './TrackTileContainer'
+import { createStyles } from './styles'
+
+const styles = StyleSheet.create({
+  skeleton: {
+    position: 'absolute',
+    top: 0
+  },
+  mainContent: {
+    flex: 1
+  },
+  metadata: {
+    flexDirection: 'row'
+  },
+  bottomButtons: {
+    position: 'absolute',
+    bottom: 0,
+    width: '100%'
+  }
+})
+
+export const TrackTileSkeleton = () => {
+  const trackTileStyles = useThemedStyles(createStyles)
+  return (
+    <TrackTileContainer>
+      <View style={styles.mainContent}>
+        <View style={styles.metadata}>
+          <View style={[trackTileStyles.imageContainer, trackTileStyles.image]}>
+            <DynamicImage
```

Comment:
would it make sense to have an explicit skeleton image component?

Reply:
Or maybe just a <View> with some styling on it versus an image in general?

Reply:
I opted for an explicit component that can be used in DynamicImage as well! It's just a LinearGradient view